media-tv/xmltv/xmltv-0.6.1.ebuild

```diff
@@ -0,0 +1,301 @@
+		dev-perl/HTML-Tree
+		dev-perl/IO-stringy
+	)
+	fr? (
+		dev-perl/DateTime
+		dev-perl/DateTime-TimeZone
+		>=dev-perl/HTML-Parser-1.27
+		dev-perl/HTML-Tree
+	)
+	huro? (
+		dev-perl/HTML-Parser
+		dev-perl/HTML-Tree
+	)
+	il? ( dev-perl/DateTime )
+	is? (
+		dev-perl/HTML-Parser
+		dev-perl/HTML-Tree
+		dev-perl/URI
+		dev-perl/XML-DOM
+		dev-perl/XML-LibXSLT
+	)
+	it? (
+		dev-perl/HTML-Parser
+		dev-perl/HTML-Tree
+		dev-perl/URI
+	)
+	na-dd? ( dev-perl/SOAP-Lite )
+	na-dtv? (
+		dev-perl/DateTime
+		dev-perl/HTTP-Cookies
+		dev-perl/URI
+	 )
+	na-tvmedia? ( dev-perl/XML-LibXML )
+	pt-meo? (
+		dev-perl/DateTime
+		dev-perl/XML-LibXML
+	)
+	pt-vodafone? (
+		dev-perl/DateTime
+		dev-perl/URI
+		dev-perl/XML-LibXML
+	)
+	se-swedb? (
+		dev-perl/HTTP-Cache-Transparent
+		dev-perl/IO-stringy
+		dev-perl/XML-LibXML
+	)
+	se-tvzon? (
+		dev-perl/XML-LibXML
+		dev-perl/IO-stringy
+		dev-perl/HTTP-Cache-Transparent
+	)
+	tr? (
+		dev-perl/DateTime
+		dev-perl/HTTP-Cache-Transparent
+		dev-perl/HTTP-Cookies
+		dev-perl/URI
+	)
+	uk-bleb? (
+		dev-perl/Archive-Zip
+		dev-perl/IO-stringy
+	)
+	uk-tvguide? (
+		dev-perl/DateTime
+		dev-perl/HTML-Tree
+		dev-perl/HTTP-Cache-Transparent
+		dev-perl/HTTP-Cookies
+		dev-perl/URI
+	)
+	zz-sdjson? (
+		>=dev-lang/perl-5.16
+		dev-perl/DateTime
+		virtual/perl-Digest-SHA
+		dev-perl/HTTP-Message
+		dev-perl/LWP-Protocol-https
+		dev-perl/Try-Tiny
+	)
+	zz-sdjson-sqlite? (
+		>=dev-lang/perl-5.16
+		dev-perl/DateTime
+		dev-perl/DateTime-Format-ISO8601
+		dev-perl/DateTime-Format-SQLite
+		dev-perl/DateTime-TimeZone
+		dev-perl/DBD-SQLite
+		dev-perl/DBI
+		virtual/perl-Digest-SHA
+		dev-perl/File-HomeDir
+		dev-perl/File-Which
+		dev-perl/List-MoreUtils
+		dev-perl/LWP-UserAgent-Determined
+	)
+	tv-check? (
+		dev-perl/Tk
+		dev-perl/Tk-TableMatrix
+	)
+	tv-combiner? ( dev-perl/XML-LibXML )
+	tv-pick-cgi? ( dev-perl/CGI )
+"
+
+PREFIX="/usr"
+
+pkg_setup() {
+	# Uses Data::Manip in various places which can fail
+	# if TZ is still set to Factory as it is in stock gentoo
+	# install media
+	export TZ=UTC
+}
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e "s:\$VERSION = '${PV}':\$VERSION = '${PVR}':" \
+		-e "/^@docs/s:doc/COPYING ::" \
+		Makefile.PL || die
+}
+
+src_configure() {
+	make_config() {
+		# Never except default configuration
+		echo "no"
+
+		# Enable Agentina
+		usex ar
+		# Enable Switzerland Search
+		usex ch-search
+		# Enable Denmark
+		usex dk
+		# Enable Latin America
+		usex dtvla
+		# Enable Europe (DEPRECATED, xmltv.se / dotmedia)
+		usex eu-dotmedia
+		# Enable EU epgdata.com including fanart
+		usex eu-epgdata
+		# Enable Europe (xmltv.se / xmltvse)
+		usex eu-xmltvse
+		# Enable Finland
+		usex fi
+		# Enable Swedish listings in Finland
+		usex fi-sv
+		# Enable France
+		usex fr
+		# Enable Hungary and Romania
+		usex huro
+		# Enable Israel
+		usex il
+		# Enable Iceland
+		usex is
+		# Enable Italy
+		usex it
+		# Enable Italy from DVB-S stream
+		echo "no" # missing Linux::DVB
```

Comment:
So this does nothing?

Reply:
Linux::DVB perl module not available in gentoo tree. Should I go ahead and enable based on flag and let it fail when run? This would give user an indication that Linux::DVB needs to be installed manually.

Reply:
Nah, its not acceptable for any USE flags to implicitly require you to invent your own way of solving a problem for portage.

I'd argue to disable the functionality entirely ( as you seem to have ), and people who want it can file a bug.

If you want to support it, the only right option is adding it to tree.

If the functionality can be obtained "by magic" by happening to install Linux::DVB *afterwards*, then, assuming its in ::gentoo, you can suggest users install it in an einfo message somewhere, but doing that or not doing that can be a tricky decision

Reply:
I'll leave it disabled entirely until I get a bug report. I found Linux::DVB in https://gentoo.zugaina.org/Overlays/perl-experimental-snapshots . It is only keyworded ~amd64 ~x86.